parley/src/layout/editor.rs

```diff
@@ -1005,11 +1022,15 @@ where
             x_offset,
             y_offset,
         );
-        if let Some(selection) = self
-            .selection
-            .to_access_selection(&self.layout, &self.layout_access)
-        {
-            node.set_text_selection(selection);
+        if self.compose.as_ref().map(|c| c.show_cursor).unwrap_or(true) {
+            if let Some(selection) = self
+                .selection
+                .to_access_selection(&self.layout, &self.layout_access)
+            {
+                node.set_text_selection(selection);
+            }
+        } else {
+            node.clear_text_selection();
```

Comment:
Is clearing the text selection, in response to the IME hiding the cursor, the right thing to do? 

(I also wonder (unrelated to this PR), whether doing nothing when `to_access_selection` is `None` is right, though it appears that only happens when in a strange state.)